Maths/Volume.js

```diff
@@ -112,6 +113,20 @@ const volHemisphere = (radius) => {
   return (2.0 * Math.PI * radius ** 3) / 3.0
 }
 
+/*
+  Calculate the volume for a Torus
+  Reference: https://en.wikipedia.org/wiki/Torus
+  return (PI * innerRadius^2) * (2 * PI * outerRadius)
```

Comment:
Please consistently use `inner`/`outer` radius (you're calling it `minor`/`major` below)